src/components/menu-bottom/MenuBottom.vue

```diff
@@ -1,19 +1,73 @@
 <template>
   <div class="menu-bottom">
-    <h3>Menu bottom</h3>
+    <div class="buttons">
+      <b-tooltip
+        v-for="(toolData, index) in toolDataList"
+        :key="index + 'toolData'"
+        :label="toolData.label"
+        :delay="500"
+      >
+        <b-button
+          :type="toolSelectedIndex === index ? 'is-primary' : 'is-light'"
+          :icon-left="toolData.icon"
+          data-test="menu-bottom--tool-button"
+          @click="setTool(index)"
+        />
+      </b-tooltip>
+    </div>
   </div>
 </template>
 
 <script lang="ts">
 import Vue from 'vue';
+import BaseTool, { ToolConstructor } from '@/tools/BaseTool';
+import ToolPanZoom from '../../tools/ToolPanZoom';
```

Comment:
Is there a reason we do some as `@` and others as `../../`?

Reply:
Yeah, this should be `@`... I'll look into a lint rule that catches this after this PR.